### PR TITLE
Remove find_graphviz check

### PR DESCRIPTION
**Pull request by MoyanZitto**
Since this method has been removed in pydot,  this find_graphviz() check would raise an exception in most cases, it's time to remove it. And if someone continue to use the old version pydot, which is the minority cases, he should update it.

### CODE REVIEW

File: keras/utils/vis_utils.py [hunk 1]
```diff
@@ -32,7 +26,6 @@ def model_to_dot(model, show_shapes=False, show_layer_names=True):
     from ..layers.wrappers import Wrapper
     from ..models import Sequential
 
-    _check_pydot()
     dot = pydot.Dot()
```
Comment (fchollet) [inline keras/utils/vis_utils.py]: Now this line triggers an exception.

Comment (MoyanZitto) [inline keras/utils/vis_utils.py]: That's strange, it turns out no exception been raised in my computer. And I am pretty sure my keras version is up-to-date, I installed it from source code just now...

Comment (fchollet) [inline keras/utils/vis_utils.py]: It triggers an exception if `pydot` is None.

**Comment by MoyanZitto** [comment]
There are some connections error in my computer. I will reopen this PR after fixed problems tomorrow

**Comment by MoyanZitto** [comment]
There are some connections error in my computer. I will reopen this PR after fixed problems tomorrow

**Comment by MoyanZitto** [comment]
There are some connections error in my computer. I will reopen this PR after fixed problems tomorrow

**Comment by MoyanZitto** [comment]
There are some connections error in my computer. I will reopen this PR after fixed problems tomorrow

**Comment by MoyanZitto** [comment]
There are some connections error in my computer. I will reopen this PR after fixed problems tomorrow

**Pull request closed without merging**